store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9802,4 +9802,37 @@ TODO: delete them permanently from here
     <defaultCOSValue>TRUE</defaultCOSValue>
     <desc>Whether Powerpaste preference is enabled by user or admin</desc>
 </attr>
+
+<attr id="3086" name="zimbraFeatureChangeUseImapEnabled" type="boolean" cardinality="single" optionalIn="account,cos" flags="accountInfo,accountInherited,domainAdminModifiable" since="8.8.15">
```

Comment:
Is it okay to set `since` to `8.8.15` or should it be `9.0.0`? Other attributes as well.

Reply:
please make attribute name more intuitive  zimbraFeatureChangeUseImapEnabled 
**zimbraFeatureImapEnabled**   would be a better choice.

Reply:
Removed.